kotlin/perform-spatial-operations/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Curious that kotlin and android have different directoy names!

Reply:
sorry, this was the source of my confusion about the proper name for the java sample. i'm really sorry to make the same mistake more than one or two times albeit for different reasons..

Reply:
What else are reviews for! I've been doing the same 😅